tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/ParseFactoryTest.java

```diff
@@ -0,0 +1,49 @@
+package com.google.cloud.bigquery.utils.queryfixer;
+
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.apache.calcite.sql.parser.SqlParser;
+import org.apache.calcite.sql.parser.impl.SqlParserImpl;
+
+import com.google.cloud.bigquery.utils.queryfixer.exception.ParserCreationException;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class ParseFactoryTest {
+
+  @Test
+  public void getParser() throws SqlParseException {
```

Comment:
Can we add a case for a slightly more complicated query?

Reply:
A new case is added.